server/service/integration_core_test.go

```diff
@@ -1847,6 +1847,8 @@ func (s *integrationTestSuite) TestGlobalPoliciesProprietary() {
 	require.NotNil(t, mgpResp.Policy.Resolution)
 	assert.Equal(t, "some global resolution updated", *mgpResp.Policy.Resolution)
 	assert.Equal(t, "darwin", mgpResp.Policy.Platform)
+	assert.Equal(t, uint(0), mgpResp.Policy.FailingHostCount)
+	assert.Equal(t, uint(0), mgpResp.Policy.PassingHostCount)
```

Comment:
One last check.

Check the same thing (`FailingHostCount` and `PassingHostCount`) in line 1861 the following view uses. the `GET /api/latest/fleet/policies/{id}` API:
![Screenshot 2023-10-30 at 19 39 11](https://github.com/fleetdm/fleet/assets/2073526/5e0f0bcd-6792-4bc6-961d-e66df6f81bd3)